packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -40,1359 +40,1323 @@ import {
   InformationCircleFilledIcon,
   SubtractIcon,
 } from "../iconDictionary";
-
-export const components: ThemeOptions["components"] = {
-  MuiAlert: {
-    defaultProps: {
-      iconMapping: {
-        error: <AlertTriangleFilledIcon />,
-        info: <InformationCircleFilledIcon />,
-        success: <CheckCircleFilledIcon />,
-        warning: <AlertTriangleFilledIcon />,
+import { DesignTokensOverride } from ".";
+
+export const components = (
+  odysseyTokens: DesignTokensOverride
+): ThemeOptions["components"] => {
+  return {
+    MuiAlert: {
+      defaultProps: {
+        iconMapping: {
+          error: <AlertTriangleFilledIcon />,
+          info: <InformationCircleFilledIcon />,
+          success: <CheckCircleFilledIcon />,
+          warning: <AlertTriangleFilledIcon />,
+        },
       },
-    },
-    styleOverrides: {
-      root: ({ ownerState, theme }) => ({
-        padding: theme.spacing(4),
-        gap: theme.spacing(4),
-        color: theme.palette.text.primary,
-        ...(ownerState.severity && {
-          backgroundColor: theme.palette[ownerState.severity].lighter,
-          borderColor: theme.palette[ownerState.severity].light,
-        }),
-        ...(ownerState.variant === "banner" && {
-          position: "relative",
-          justifyContent: "center",
-          alignItems: "center",
-          borderWidth: 0,
-          borderRadius: 0,
-        }),
-        ...(ownerState.variant === "infobox" && {
-          borderStyle: theme.mixins.borderStyle,
-          borderWidth: theme.mixins.borderWidth,
-          borderRadius: theme.mixins.borderRadius,
-          "&:not(:last-child)": {
-            marginBottom: theme.spacing(4),
-          },
+      styleOverrides: {
+        root: ({ ownerState, theme }) => ({
+          padding: theme.spacing(4),
+          gap: theme.spacing(4),
+          color: theme.palette.text.primary,
+          ...(ownerState.severity && {
+            backgroundColor: theme.palette[ownerState.severity].lighter,
+            borderColor: theme.palette[ownerState.severity].light,
+          }),
+          ...(ownerState.variant === "banner" && {
+            position: "relative",
+            justifyContent: "center",
+            alignItems: "center",
+            borderWidth: 0,
+            borderRadius: 0,
+          }),
+          ...(ownerState.variant === "infobox" && {
+            borderStyle: theme.mixins.borderStyle,
+            borderWidth: theme.mixins.borderWidth,
+            borderRadius: odysseyTokens.BorderRadiusBase,
```

Comment:
I made one change to make use of the odysseyTokens. @edburyenegren-okta

Reply:
Awesome - good to have a test case in here. This might be failing because it should be `odysseyTokens.BorderRadiusMain`

Reply:
yes, just fixed it. I had the old tokens in my build dir